skbio/alignment/_tabular_msa.py

```diff
@@ -259,6 +261,21 @@ def __init__(self, sequences, metadata=None, positional_metadata=None,
             self, positional_metadata=positional_metadata)
 
     @experimental(as_of='0.4.0-dev')
+    def __repr__(self):
+        pep8_line_length_limit = 79
+        length_taken_by_docstring_indent = 8
+        width = pep8_line_length_limit - length_taken_by_docstring_indent
+        return _TabularMSAReprBuilder(
+            seq=self,
+            width=width,
+            indent=4).build()
+
+    @experimental(as_of='0.4.0-dev')
+    def _repr_stats(self):
+        return [("Sequence Count", str(self.shape.sequence)),
```

Comment:
lowercase these for consistency with `Sequence` stats:

"sequence count"
"position count"